php/Admin/EditBlock.php

```diff
@@ -111,51 +111,32 @@ public function enqueue_assets() {
 					[
 						'postType'     => get_post_type(),
 						'postId'       => get_the_ID(),
-						'settings'     => [],
-						'initialEdits' => $this->get_initial_edits(),
+						'settings'     => [
+							'titlePlaceholder'   => __( 'Block title', 'genesis-custom-blocks' ),
+							'richEditingEnabled' => false,
+						],
+						'initialEdits' => null,
 						'controls'     => genesis_custom_blocks()->block_post->get_controls(),
 					]
 				)
 			),
 			'before'
 		);
 
+		$edit_block_style_path = 'css/edit-block.css';
 		wp_enqueue_style(
 			self::STYLE_SLUG,
-			$this->plugin->get_url( 'css/edit-block.css' ),
+			$this->plugin->get_url( $edit_block_style_path ),
 			[ 'wp-components' ],
-			$this->plugin->get_version()
+			filemtime( $this->plugin->get_path( $edit_block_style_path ) )
 		);
 
 		// @todo: get only the style rules that are needed, and add them to a CSS file in this plugin.
 		wp_enqueue_style(
 			self::TAILWIND_SLUG,
-			'https://unpkg.com/tailwindcss@1.9.6/dist/tailwind.min.css',
+			$this->plugin->get_url( 'css/tailwind.min.css' ),
 			[],
 			$this->plugin->get_version()
 		);
 	}
-
-	/**
-	 * Gets the initial edits, forked from Core.
-	 *
-	 * @return array|null The initial edits.
-	 */
-	public function get_initial_edits() {
```

Comment:
This probably doesn't apply to us. 

If it's a new post, we have defaults ready to add, like a title of 'New Field' if they don't enter one.